pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/RealtimeSegmentDataManager.java

```diff
@@ -284,6 +284,7 @@ public void deleteSegmentFile() {
 
   private Thread _consumerThread;
   private final int _partitionGroupId;
+  private final int _streamPatitionGroupId;
```

Comment:
Please add javadoc explaining the difference between this id vs. _partitionGroupId.